pkg/reactor/content_processor_test.go

```diff
@@ -477,9 +477,15 @@ type testResourceHandler struct {
 func (rh *testResourceHandler) Accept(uri string) bool {
 	return true
 }
-func (rh *testResourceHandler) ResolveNodeSelector(ctx context.Context, node *api.Node, excludePaths []string, frontMatter map[string]interface{}, excludeFrontMatter map[string]interface{}, depth int32) error {
-	return nil
+
+func (rh *testResourceHandler) ResolveNodeSelector(ctx context.Context, node *api.Node, excludePaths []string, frontMatter map[string]interface{}, excludeFrontMatter map[string]interface{}, depth int32) ([]*api.Node, error) {
+	return nil, nil
+}
+
+func (rh *testResourceHandler) ResolveModule(ctx context.Context, uri string) (*api.Documentation, error) {
```

Comment:
Term-wise I'd prefer not to introduce new ones that will need further explanations. Maybe stick to _manifest_ or _structure_

Reply:
Done